lambdas/shared/get-all-usage-plans.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
So, contrary to the title, I'm pretty sure none of these (things in lambdas/shared) are actually shared / used... I don't think there's any action to take on this CR, but [here's an issue for addressing it later](https://github.com/awslabs/aws-api-gateway-developer-portal/issues/287).

Reply:
Noted.